resources/web/docs_js/index.js

```diff
@@ -194,6 +206,32 @@ $(function() {
   init_sense_widgets();
   init_console_widgets();
   init_kibana_widgets();
+  $("div.ess_widget").each(function() {
+    const div         = $(this),
+          snippet     = div.attr('data-snippet'),
+          consoleText = div.prev().text() + '\n';
+
+    return mount(div, ConsoleWidget, {
+      setting: "ess",
+      url_label: 'Enter the URL of Elastic Cloud',
```

Comment:
I would change to `Enter the endpoint URL of the Elasticsearch Service`.

Rationale: 
* I made a global comment earlier about making sure we use the right service name ... just to repeat that comment here: we definitely want to use `Elasticsearch Service`, as the Elastic Cloud umbrella name now encompasses all the SaaS offerings we have (including things like Elastic App Search Service). 
* Also, I would change `URL` --> `endpoint URL`, as that is the terminology we use in the ESS UI.